### ISSUE TITLE
NINA: Switch to Random Forest Classifier

### ISSUE THREAD

**Issue by ArniGoyal** [GDG-IGDTUW/AI-ML-1#19]
Our current baseline uses Logistic Regression. While efficient, a tree-based ensemble model like Random Forest might better capture non-linear relationships in the news data.

**Tasks:** 

- Import RandomForestClassifier from sklearn.ensemble.

- Train a model using the existing embeddings.

- Print a comparison of the Accuracy and F1-score against the Logistic Regression baseline.

**Comment by anoushka1405** [comment]
@ArniGoyal Please assign this issue to me

**Comment by anoushka1405** [comment]
@ArniGoyal Please assign this issue to me

**Comment by anoushka1405** [comment]
@ArniGoyal Please review my PR and let me know is any changes are needed. Thank You!
